proposals/stage-1/CODE_OF_CONDUCT/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -1,15 +1,16 @@
 # Foundation Code of Conduct Requirements
 
 It is important that there be a Code of Conduct (CoC) which
-is documented and enforced for the OpenJS Foundation and the
+is documented and enforced for the OpenJS Foundation, 
+hereinafter referred to as the "Foundation", and the
```

Comment:
Maybe this?:

```markdown
is documented and enforced for the OpenJS Foundation (_the Foundation_) 
```

Reply:
The word "hereinafter" sounds unusual but IMO writing out that in the document we're subsequently calling the OpenJS Foundation just "Foundation" is worth the extra words for to avoid any ambiguity.